src/components/PostDetails.tsx

```diff
@@ -65,28 +34,14 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
         postId: post.id,
       });
 
-      setComments(
-        currentComments => [...currentComments, newComment],
-      );
-
-      // setComments([...comments, newComment]);
-      // works wrong if we wrap `addComment` with `useCallback`
-      // because it takes the `comments` cached during the first render
-      // not the actual ones
+      dispatch(setComments([...comments, newComment]));
```

Comment:
Looks like we should create addComment action to add new comment
```suggestion
      dispatch(addComment(newComment));
```